docs/installation/linux/fedora.md

```diff
@@ -12,80 +12,96 @@ weight=-3
-using Fedora-managed packages, consult your Fedora release documentation for
-information on Fedora's Docker support.
+Docker is supported on Fedora version 22, 23, and 24. This page instructs you to
+install using Docker-managed release packages and installation mechanisms. Using
+these packages ensures you get the latest release of Docker. If you wish to
+install using Fedora-managed packages, consult your Fedora release documentation
+for information on Fedora's Docker support.
 
 ## Prerequisites
 
-Docker requires a 64-bit installation regardless of your Fedora version. Also, your kernel must be 3.10 at minimum. To check your current kernel
-version, open a terminal and use `uname -r` to display your kernel version:
+Docker requires a 64-bit installation regardless of your Fedora version. Docker
+requires that your kernel must be 3.10 at minimum, which Fedora runs.
 
-    $ uname -r
-    3.19.5-100.fc21.x86_64
+To check your current kernel version, open a terminal and use `uname -r` to
+display your kernel version:
 
-If your kernel is at an older version, you must update it.
+```bash
+$ uname -r
+3.19.5-100.fc21.x86_64
+```
 
-Finally, is it recommended that you fully update your system. Please keep in
-mind that your system should be fully patched to fix any potential kernel bugs. Any
-reported kernel bugs may have already been fixed on the latest kernel packages
+If your kernel is at an older version, you must update it.
 
+Finally, it is recommended that you fully update your system. Please keep in
+mind that your system should be fully patched to fix any potential kernel bugs.
+Any reported kernel bugs may have already been fixed on the latest kernel
+packages.
 
-## Install
+## Install Docker Engine
 
-There are two ways to install Docker Engine.  You can install with the `dnf` package manager. Or you can use `curl` with the  `get.docker.com` site. This second method runs an installation script which also installs via the `dnf` package manager.
+There are two ways to install Docker Engine.  You can [install using the `dnf`
+package manager](#install-with-dnf). Or you can use `curl` [with the  `get.docker.com`
+site](#install-with-the-script). This second method runs an installation script
+which also installs via the `dnf` package manager.
 
 ### Install with DNF
 
 1. Log into your machine as a user with `sudo` or `root` privileges.
 
-2. Make sure your existing dnf packages are up-to-date.
+2. Make sure your existing packages are up-to-date.
 
-		$ sudo dnf update
+    ```bash
+    $ sudo dnf update
+    ```
 
-3. Add the yum repo yourself.
+3. Add the yum repo.
 
-        $ sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
-        [dockerrepo]
-        name=Docker Repository
-        baseurl=https://yum.dockerproject.org/repo/main/fedora/$releasever/
-        enabled=1
-        gpgcheck=1
-        gpgkey=https://yum.dockerproject.org/gpg
-        EOF
+    ```bash
+    $ sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
```

Comment:
This seems a little complicated. Aren't people used to editing these files using a text editor? Copy/paste?